dcc-download-server/src/main/java/org/icgc/dcc/download/server/io/GzipStreamer.java

```diff
@@ -184,7 +184,10 @@ private long getCurrentHeaderLength() {
   }
 
   private void checkArguments() {
-    checkArgument(!downloadFiles.isEmpty());
+    if (downloadFiles.isEmpty()) {
+      Responses.throwNotFoundException("Job has no files to download",
```

Comment:
Use a static import for `throwNotFoundException`

Should be `ctrl` + `shift` + `m` in eclipse if I recall correctly.